jdaviz/app.py

```diff
@@ -2391,6 +2391,7 @@ def _expose_meta(key):
             'locked': False,
             'ndims': data.ndim,
             'type': typ,
+            'has_wcs': wcsaxes is not None,
```

Comment:
is this sufficient or do we need to do something more advanced?

Reply:
`wcsaxes` is: 

https://github.com/bmorris3/jdaviz/blob/c33b10efc7a7bf4f1841b9b4af0c349e3aa92cf0/jdaviz/app.py#L2340

and I expect that this keyword is not available in every data structure that we translate to `Data`. For example, if you make load this  `NDData`:
```python
from astropy.nddata import NDData
from astropy.wcs import WCS

ndd = NDData(np.zeros((10, 10)), wcs=WCS())
```
it doesn't have any metadata, so this probably isn't sufficient.

Reply:
I saw different logic somewhere else - can we consolidate these in a function somewhere so we're consistent?  What is the appropriate logic to check "should this data support being plotted while WCS-linked"?

Reply:
moved to utils and used in all places I could find in https://github.com/bmorris3/jdaviz/pull/14/commits/b52ac5d80e59511d69dd2164f8a8df1c0473b83e